design/temp.txt

```diff
@@ -0,0 +1 @@
+Nothing at the moment
```

Comment:
Need to remove this file. Design for the lab will be put into the lab repo once created.